public/scripts/app.js

```diff
@@ -47,66 +41,26 @@ $('ul.nav li').click(function(){
          `
        );
      }
-   }
-   else if(($(this).context.textContent)==="My Work")
-   {
-     //projects section
-
+    } else if(($(this).context.textContent)==="My Work") {
      $("#addProject").fadeIn();
      $("#addVacation").fadeOut();
-
-     console.log("Projects section");
-     $.ajax({
-       method:"GET",
-       url:'/api/profile/projects',
```

Comment:
You don't need this route. Your `/api/profile` endpoint already gave you all your projects, because your projects were embedded. 

This code is also adding nothing to your page, because you're adding all projects to the page using the `/api/profile` AJAX call.